ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -76,12 +76,17 @@ class ClientQueryViewController: UITableViewController, Themeable, UIDropInterac
 		query.addObserver(self, forKeyPath: "state", options: .initial, context: nil)
 		core?.start(query)
 
-		var title = (query.queryPath as NSString?)!.lastPathComponent
+		var title : String?
 
-		if title == "/", let shortName = core?.bookmark.shortName {
-			title = shortName
-		}
+		if let queryTitle = (query.queryPath as NSString?)?.lastPathComponent {
+			title = queryTitle
 
+			if title == "/" {
```

Comment:
@felix-schwarz I think having some category method on `OCPath` (`isRoot`) would be nicer than to check if it is "/"

Reply:
Yep. Thought about this yesterday, too. I'll add that property in the next SDK update.

Reply:
@felix-schwarz ok, cool!